.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Ticket
+<!---
+Attach a link to the ticket. leave nothing other than this hyperlinked task here.
+-->
+[Task](https://arceeai.slack.com/lists/XXXXXXX/YYYYYYY?record_id=ZZZZZZZZ)
```

Comment:
it looks like we're use slack for platform and potentially jira for research. i chose slack prematurely, but open to swapping this out

Reply:
@mryave since this one is public facing/OS we might want to do PR open for community 

Julien was forking and PR'ing in like we might want users doing here (eventually)